convert-hf-to-gguf.py

```diff
@@ -2793,6 +2799,26 @@ def parse_args() -> argparse.Namespace:
         "--verbose", action="store_true",
         help="increase output verbosity",
     )
+    parser.add_argument(
+        "--split", action="store_true",
+        help="split the converted model into multiple files",
+    )
+    parser.add_argument(
+        "--split-max-tensors", type=int,
+        help="max tensors in each split",
+    )
+    parser.add_argument(
+        "--split-max-size", type=str,
+        help="max size per split N(M|G)",
+    )
+    parser.add_argument(
+        "--dry-run", action="store_true",
+        help="only print out a split plan and exit, without writing any new files",
+    )
+    parser.add_argument(
+        "--small-first-shard", action="store_true",
+        help="do not add tensors to the first shard (disabled by default)",
```

Comment:
`shard` vs `split`, might not be obvious that this is related to splits. Should the other options be renamed to `--shard-max-size` and `--shard-max-tensors`?

(You can answer "no" too)

Reply:
Renaming for consistency with examples/gguf-split is probably the way to go here.